akka-docs/src/main/paradox/stream/operators/ActorSource/actorRefWithBackpressure.md

```diff
@@ -34,7 +34,7 @@ See also:
 
 ## Example
 
-With `actorRefWithBackpressure` two actors get into play: one which feeds the stream and is materialized when the stream runs, and one which is provided by us and gets the ack signal when an element is emitted into the stream.
+With `actorRefWithBackpressure` two actors get into play: one which feeds the stream and is materialized when the stream runs, and one which is provided by us which gets the ack signal when an element is emitted into the stream.
```

Comment:
Maybe fewer `which`s and even a numbered list could be helpful:

```suggestion
With `actorRefWithBackpressure` two actors get into play: 

1. An actor that is materialized when the stream runs. It feeds the stream.
2. An actor provided by the user. It gets the ack signal when an element is emitted into the stream.
```

In the list I also reordered the sentences to the "who provides the actor. What does the actor do" format.

Reply:
This one is good. I'll do this.